cheatsheets/Authentication_Cheat_Sheet.md

```diff
@@ -341,6 +341,8 @@ UAF takes advantage of existing security technologies present on devices for aut
 
 U2F augments password-based authentication using a hardware token (typically USB) that stores cryptographic authentication keys and uses them for signing. The user can use the same token as a second factor for multiple applications. U2F works with web applications. It provides **protection against phishing** by using the URL of the website to look up the stored authentication key.
 
+**FIDO2**: FIDO2 and WebAuthn, encompassing previous standards (UAF/U2F), form the foundation of modern **Passkeys** technology. Passkeys enable users to securely log in using their biometric data or device locks, synchronizing across devices, password-free, and phishing-resistant. This technology is widely supported by major platforms.
```

Comment:
"biometric data or device locks" - biometric isn't always included, and I'm not sure what device locks mean. Can this be clarified please?
Also "synchronizing across devices" is not always present IIRC.